Assets/MixedRealityToolkit.Extensions/SpectatorView/Scripts/StateSynchronization/Observer.cs

```diff
@@ -0,0 +1,228 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Extensions.Experimental.Socketer;
+using System.Collections.Generic;
+using System.IO;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Extensions.Experimental.SpectatorView
+{
+    /// <summary>
+    /// This class observes changes and updates content on a spectator device.
+    /// </summary>
+    public class Observer : Singleton<Observer>
```

Comment:
>Observer [](start = 17, length = 8)

Similar comment here to Broadcaster: maybe make this ObserverNetworkManager or SynchronizedStateObserver